src/optimistic-oracle-v2/implementation/ManagedOptimisticOracleV2.sol

```diff
@@ -205,52 +237,46 @@ contract ManagedOptimisticOracleV2 is
 
     /**
      * @notice Set the proposal bond associated with a price request.
+     * @dev This would also override any subsequent calls to setBond() by the requester.
```

Comment:
Maybe we could clarify that this function can be used to set a bond for requests that don’t exist yet and that this is actually one of its intended uses.